CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# dbt_twilio_source v0.1.1
```

Comment:
Suggestion to bump this to v0.2.0 so we avoid unexpected breaks with the dbt_utils.star update.

Reply:
updated